### PR TITLE
dnd-2 (Create a list of Player Cards)

### PR DESCRIPTION
**Pull request by nilz3ro**
# dnd-2

### About These Changes

* Introduces `PlayerCard` and `CardsContainer` components.

### CODE REVIEW

File: src/CardContainer.js [hunk 1]
```diff
@@ -1,26 +1,82 @@
...
+          id: 3
+        }
+      ]
+    };
+  }
 
-CardContainer.propTypes = {
-  players: PropTypes.array
-};
+  handleEditClick = player => {
```
Comment (nilz3ro) [inline src/CardContainer.js:50]: @spencermendoza nice naming here.

File: src/CardContainer.js [hunk 2]
```diff
@@ -1,26 +1,82 @@
...
+        <Box display="flex" flexDirection="column" justifyContent="center">
+          {this.state.players.map(player => (
+            <PlayerCard
+              player={player}
+              key={player.id}
+              onEditClick={p => this.handleEditClick(p)}
+            />
+          ))}
+        </Box>
+        <PlayerFormDialog
```
Comment (nilz3ro) [inline src/CardContainer.js:72]: @spencermendoza This is another good name. Nice job :sparkles:

**Review by nilz3ro — COMMENTED** [review]
Done with dnd-2. @spencermendoza could you check out this PR?

File: src/CardContainer.js (hunk 1 above)
Comment (spencermendoza) [reply src/CardContainer.js:50]: thanks it was 100% my idea :sparkles:

**Review by spencermendoza — APPROVED** [review]

**Pull request merged**